src/main/java/org/broadinstitute/hellbender/tools/spark/utils/SparkUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.tools.spark.utils;
+
+import org.apache.log4j.Logger;
+import org.apache.spark.broadcast.Broadcast;
+
+public class SparkUtils {
```

Comment:
There's already a `SparkUtils.java` living in `org.broadinstitute.hellbender.utils.spark`. Maybe this can be moved there...

Reply:
Thanks.  Moved it there, as you suggested.